CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+  contribution.
+- Issues labeled https://github.com/openinf/.github/labels/wip are a _work in
+  progress_ and generally not available, but may be available if there has been
+  no activity on the issue or related PR for over a week.
+
+Once you have identified an issue you would like to work on, follow these steps:
+
+1.  Comment on it and say you would like to work on that issue.
+2.  Wait for someone to confirm that you may work on the issue before writing
+    any code. The person who confirms will add a
+    https://github.com/openinf/.github/labels/wip label to the issue to indicate
+    that the issue has been assigned.
+3.  Once the issue has been labeled as a _work in progress_, write your code and
+    submit your PR.
+4.  Wait for code review and address any issues raised as soon as you can.
+
+Even if you are not done with the issue, create a [draft pull request][] and
+push your code [early and often][]. If we haven’t heard from you in over a week
+and someone else expresses interest in that issue, we may approve the new
+person’s work.
+
+### Opening a new issue
+
+If you want to work on something that there is no GitHub issue for, then propose
+the change by opening a new GitHub issue associated with the respective
+repository and propose your change there. Be sure to include implementation
+details and the rationale for the proposed change.
+
+## Pull requests
+
+If you’d like to propose and collaborate on changes, open a [pull request][]!
+
+Here are a few things you can do that will increase the likelihood of having
+your pull request merged:
+
+- Follow standards for style and code quality.
+- Write tests when applicable.
+- Keep your change as focused as possible. If there are multiple changes you
+  would like to make that are not dependent upon each other, consider submitting
+  them as separate pull requests.
+- Write a [good commit message][].
+
+Contributions to this project are [released][contrib-license] to the public
+under the project’s open source license. The license for a project is located in
+a file named `LICENSE` in the root directory of the repository.
+
+## Project documentation
+
+The `README.md` in the root of the repository should contain or link to project
+documentation. If you cannot find the documentation you’re looking for, please
+file a GitHub issue with details of what you’d like to see documented.
+
+## Questions or thoughts?
+
+Support requests (e.g., asking questions) or feedback (e.g., constructive
+criticism) may be directed to the [`@openinf`][twitter-account] support account
+on Twitter.
+
+## Additional resources
+
+- [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/)
+- [Using Pull Requests](https://help.github.com/articles/about-pull-requests/)
+- [GitHub Help](https://help.github.com)
+
+## Code of Conduct
+
+By participating in this project, you are expected to uphold our [Code of
+Conduct][code_of_conduct].
+
+## Developer’s Certificate of Origin 1.1
```

Comment:
This touches on a key point related to "commit message validation" not yet covered by https://github.com/openinf/openinf.github.io/issues/32, namely the `Signed-off-by: ` trailer.